index.html

```diff
@@ -20,7 +20,7 @@
   <div id="get-ready">
     <h1>Get ready to Spot the Surveillance...</h1>
   </div>
-    <a-scene>
+    <a-scene loadscene>
```

Comment:
this is an interesting way to go about it. I wonder what modern A-Frame projects do.

Reply:
Yup, pretty normal in many examples I've seen. Makes sense since the component is more general/global functions.